### ISSUE TITLE
Call logs are not backing up on LG G4 (Dual SIM),  Android version 5.1, with XOAuth or IMAP. 

### ISSUE THREAD

**Issue by Vachha** [jberkel/sms-backup-plus#611]
First I must say what a excellent piece of software this is, my whole family have been using it for years on various HW and Android versions. This is the only issue I have had :-)

I restored my old phone _(Huawei G700-U10 Dual SIM model)_ Call log and SMS backup successfully to the LG G4.

SMS is working fine both in XOAuth and IMAP modes.

I have done some experimentation including:
- Enabling SIM1 and SIM2 in turn and simultaneously (in both XOAuth and IMAP mode).
- Using IMAP (as I am the server admin and can get provide logs) - no issue with this SMS backup working fine.
- Sometimes after swapping SIMS and disabling SIM1 backup is sucessful the first time only.
- Tried the latest beta version - same result.

Symptoms are (both both XOAuth and IMAP mode):
- Brief message saying correct number of records to be updated is displayed.
- Backup runs
- Exits without error and displays "There were no items to backup." 

_Thank you._

Sanitized logs follow below:

Android Log
###### 

06-11 13:45 Starting backup (MANUAL)
06-11 13:45 Backing up (0 SMS, 0 MMS, **2 call log**)
06-11 13:45 Using config: BackupConfig{imap=BackupImapStore{uri=imap+ssl://PLAIN:XXXXXXXXXXXXXX@xxxx.com.hk:993}, skip=false, currentTry=0, maxItemsPerSync=-1, groupToBackup=ContactGroup{_id=-1, type=EVERYBODY}, backupType=MANUAL, debug=true, typesToBackup=[SMS, CALLLOG]}
06-11 13:45 BackupState{currentSyncedItems=0, **itemsToSync=0**, backupType=MANUAL, exception=null, state =FINISHED_BACKUP}
06-11 13:45 Backup finished

IMAP Server Log
###### 

Nov  6 13:45:15 xxxx dovecot: imap-login: Login: user=<XXXXXXXXXXXXXX>, method=PLAIN, rip=192.168.XXX.XXX, lip=192.168.YYY.YYY, mpid=26658, TLS, session=<iFFyv9gjoQDAqNKm>
Nov  6 13:45:15 xxxx dovecot: imap(XXXXXXXXXXXXXX): Connection closed in=42 out=628

**Comment by jberkel** [comment]
Hm, wonder if it could have something to do with the dual sim?

**Comment by Vachha** [comment]
Very strange in deed, I just looked at it again, not sure why but it seems to be working normally now.

I did have 2 SIM cards active for a month in Dec / Jan and there was also a firmware update around the end of Dec so not sure what fixed it. However looking back at the extended logs it does seem to have been working before the firmware update if synced automatically.  I suspect it may just have been an anomaly of the particular phone and firmware version.
